specs/darwin/safari_extensions.table

```diff
@@ -1,8 +1,7 @@
 table_name("safari_extensions")
 description("Safari browser extension details for all users. This table requires Full Disk Access (FDA) permission.")
 schema([
-    Column("uid", BIGINT, "The local user that owns the extension",
-      index=True),
+    Column("uid", BIGINT, "The local user that owns the extension", index=True, optimized=True),
```

Comment:
`usersFromContext` is valid for the optimization, but there are a few generation methods I found dropped results with it. The `crashes` and `browser_plugins` tables most notably. I think this is because they utilize `context.constraints["uid"].notExistsOrMatches()`, which breaks the `IN` constraint as `notExistsOrMatches` ensures all passed-in constraints match the compared value to be true.

```
if (context.constraints["uid"].notExistsOrMatches("0")) {
  process_crash_logs(kDiagnosticReportsPath, "application");
}
```

When xFilter is being ran once per constraint then it returns both the user and non-user context, but with a single xFilter call like `WHERE uid IN (0, 501)`, then only the user context is returned.